app/src/main/java/org/gnucash/android/ui/account/AccountsActivity.java

```diff
@@ -227,6 +227,7 @@ public void onCreate(Bundle savedInstanceState) {
 
         init();
 
+        getSupportActionBar().setSubtitle(BooksDbAdapter.getInstance().getActiveBookDisplayName());
```

Comment:
Might be better to create a separate pull request for this. 
Also, if possible, style the subtitle smaller so that the main title does not feel pushed up too much. 

How about the other activities in the app. Might be good to think a bit about how the book name could be integrated into them as well.